src/rpc/core_rpc_server.cpp

```diff
@@ -3416,7 +3427,7 @@ namespace cryptonote
     if (exceeds_quantity_limit(ctx, error_resp, m_restricted, req.entries.size(), COMMAND_RPC_LNS_OWNERS_TO_NAMES::MAX_REQUEST_ENTRIES))
       return false;
 
-    std::map<lns::generic_owner, size_t> owner_to_request_index;
+    std::map<size_t, size_t> owner_to_request_index;
```

Comment:
This should be an unordered_map (but keeping the `std::hash` specialization).  This isn't just an optimal data structure thing: using a `map` here has a potential (if rare) data omission on collision.  (And switching to an `unordered_map` is a nearly trivial change).

Reply:
Ok, updated